fineract-provider/src/main/resources/db/changelog/tenant/parts/0025_add_delinquency_buckets.xml

```diff
@@ -0,0 +1,205 @@
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="min_age_days" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="max_age_days" type="BIGINT">
+                <constraints nullable="true" />
+            </column>
+            <column name="created_by" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="created_on_utc" type="DATETIME">
+                <constraints nullable="false" />
+            </column>
+            <column name="version" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueComputed="NULL" name="last_modified_by" type="BIGINT"/>
+            <column defaultValueComputed="NULL" name="last_modified_on_utc" type="DATETIME"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="fineract" id="2">
+        <createTable tableName="m_delinquency_bucket">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(100)">
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="created_by" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="created_on_utc" type="DATETIME">
+                <constraints nullable="false" />
+            </column>
+            <column name="version" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueComputed="NULL" name="last_modified_by" type="BIGINT"/>
+            <column defaultValueComputed="NULL" name="last_modified_on_utc" type="DATETIME"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="fineract" id="3">
+        <createTable tableName="m_delinquency_bucket_mappings">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="delinquency_range_id" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="delinquency_bucket_id" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="created_by" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="created_on_utc" type="DATETIME">
+                <constraints nullable="false" />
+            </column>
+            <column name="version" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueComputed="NULL" name="last_modified_by" type="BIGINT"/>
```

Comment:
@josehernandezfintecheandomx last_modifiied_by and last_modified_on_utc cannot be null! Please change them!